ios/ios/SpeakerTableViewCell.swift

```diff
@@ -19,7 +19,9 @@ class SpeakerTableViewCell: UITableViewCell {
         nameLabel.text = name
         infoLabel.text = info//formatHTMLString(info)
         
-        speakerImage.kf.setImage(with: URL(string: imgUrl)!)
+        if (!imgUrl.isEmpty) {
```

Comment:
Do we need to do anything when there isnt a url? Like load a placeholder, or hide the imageview?